src/components/TransactionsTable.vue

```diff
@@ -157,75 +201,82 @@ div.row.col-12.q-mt-xs.justify-center.text-left
-            :columns="columns"
-            row-key="name"
-            flat
-            :bordered="false"
-            :square="true"
-            table-header-class="table-header"
-            v-model:pagination="paginationSettings"
-            v-model:expanded="expanded"
-            :hide-pagination="!hasPages"
-            @update:expanded='updateExpanded'
-            )
-          template( v-slot:body-cell-transaction="props")
-            q-td( :props="props" )
-              div(v-html="props.value")
-          template( v-slot:body-cell-timestamp="props")
-            q-td( :props="props" )
-              DateField( :timestamp="props.value", showAge=true )
-          template( v-slot:body-cell-action="props")
-            q-td( :props="props" )
-              div(v-html="props.value")
-          template( v-slot:body-cell-data="props")
-            q-td( :props="props"  )
-              div(v-html="props.value" :class="{'row-expanded': props.expand  }")
-              q-icon.expand-icon(v-if="checkIsMultiLine(props.value)" @click="props.expand = !props.expand" :name="props.expand ? 'expand_less' : 'expand_more'" size='.75rem')
-          template( v-slot:pagination="scope")
-            div.row.col-12.q-mt-md.q-mb-xl()
-            div.col-1(align="left")
-              q-btn.q-ml-xs.q-mr-xs.col.button-primary(
-                :disable="scope.isFirstPage"
-                @click="scope.prevPage") PREV
-            q-space
-            div.col-1(align="right")
-              q-btn.q-ml-xs.q-mr-xs.col.button-primary(
-                :disable="scope.isLastPage"
-                @click="scope.nextPage") NEXT
+        div.row.col-12.table-container
+          q-table.q-mt-lg.row.fixed-layout(
+              :rows="rows"
+              :columns="columns"
+              row-key="name"
+              flat
+              :bordered="false"
+              :square="true"
+              table-header-class="table-header"
+              v-model:pagination="paginationSettings"
+              v-model:expanded="expanded"
+              :hide-pagination="!hasPages"
+              @update:expanded='updateExpanded'
+              )
+            template( v-slot:body-cell-transaction="props")
+              q-td( :props="props" )
+                div(v-html="props.value")
+            template( v-slot:body-cell-timestamp="props")
+              q-td( :props="props" )
+                DateField( :timestamp="props.value", showAge=true )
+            template( v-slot:body-cell-action="props")
+              q-td( :props="props" )
+                div(v-html="props.value").row.justify-left
+            template( v-slot:body-cell-data="props")
+              q-td( :props="props"  )
+                div(v-html="props.value" :class="{'row-expanded': props.expand  }")
+                //- q-icon.expand-icon(v-if="checkIsMultiLine(props.value)" @click="props.expand = !props.expand" :name="props.expand ? 'expand_less' : 'expand_more'" size='.75rem')
```

Comment:
Are you sure? I disabled the expanding rows by commenting some of the html and css out, so that we can add the feature back later. If I remove these comments, should I remove the rest of the functionality as well? In other words, the logic that keeps track of the expanded rows?

Reply:
gotcha, that's fine.